src/_pytest/logging.py

```diff
@@ -183,6 +244,12 @@ def add_option_ini(option, dest, default=None, type=None, **kwargs):
         default=DEFAULT_LOG_DATE_FORMAT,
         help="log date format as used by the logging module.",
     )
+    add_option_ini(
+        "--log-auto-indent",
+        dest="log_auto_indent",
+        default=None,
+        help="auto indent multiline messages passed to the logging module. accepts true|on, false|off or an integer",
```

Comment:
```suggestion
        help="Auto-indent multiline messages passed to the logging module. Accepts true|on, false|off or an integer.",
```

The style with help messages is not really consistent, but that's more in line.. see `pytest --help`.

Reply:
done

Reply:
You missed 3/4 of my suggestions?!

Reply:
I didn't see all of the changes because the line was cut off and some of the changes were hidden behind the sideways scroll. I will do it again